src/containers/widget/index.js

```diff
@@ -45,12 +47,23 @@ class Widget extends Component {
   }
 
   componentDidMount() {
-    this.setDefaults();
+    return fetch(`${this.props.apiBaseUrl}/v1/configuration`,
+        { headers: {"Api-Token": this.props.apiToken} })
+      .then(response => {
+        if(response.ok) {
+          response.json().then(config => {
+            this.widgetConfig = config;
+            this.setDefaults(config);
```

Comment:
you are not consuming the config on `setDefaults`, maybe you should?

Reply:
thanks. yes, that's old.